diff --git a/./cairo_compile.rs b/./cairo_compile.rs
index 14c26b0..4c9f535 100644
--- a/./cairo_compile.rs
+++ b/./cairo_compile.rs
@@ -68,7 +68,7 @@ pub enum CompilationArtifacts {
 /// Returns the version of the Cairo1 compiler defined in the root Cargo.toml (by checking the
 /// package version of one of the crates from the compiler in the dependencies).
 pub fn cairo1_compiler_version() -> String {
-    let cargo_toml: CargoToml = toml::from_str(include_str!("../../../../Cargo.toml")).unwrap();
+    let cargo_toml: CargoToml = toml::from_str(include_str!("../../Cargo.toml")).unwrap();
     match cargo_toml.workspace.dependencies.cairo_lang_casm {
         DependencyValue::String(version) | DependencyValue::Object { version } => version.clone(),
     }
